modules/request-quota-increase/generate_code.rb

```diff
@@ -0,0 +1,68 @@
+#################################################################
+# REQUEST QUOTA INCREASE CODE GENERATOR
+# This script generates the code for add more resources into the
+# module. It only generates the code for the resources that
+# 'Adjustable' is true.
```

Comment:
This needs a bit of context first: i.e., that AWS requires you to pass in special codes to request quotas, that there is no central place where these are all defined, that there are many of these, we've included some in the `request-quota-increase` module already, but to add the others, this script can be used on a service-by-service basis.

Reply:
Fixed! 4b3b8e6